tests/test_StatusCodeSender.js

```diff
@@ -26,19 +26,14 @@ describe("A status code sender", function () {
 	it("gives a custom message for 400", function () {
 		const payload = {
 			errors: [
-				{message: "custom message"}
+				{message: "unexpected error"}
 			]
 		};
-		return expectedDefaultError(400, payload);
+		return expectedDefaultError(400, payload.errors[0].message);
```

Comment:
This test is passing for the wrong reasons. You should be able to pass any message in the payload and have it pass. If you change the message on line 29 to  `"custom error message from payload"` then this test fails because it won't be passing the payload to the sender. We need to pass the payload to the sender so it can parse it correctly. As this test is right now it is essentially testing the same functionality as the test on line 35

- [ ] Fix test to actually use the message from the payload